build.sh

```diff
@@ -119,8 +119,8 @@ else
   # generated from https://github.com/ray-project/arrow-build from
   # the commit listed in the command.
   $PYTHON_EXECUTABLE -m pip install \
-      --target=$ROOT_DIR/python/ray/pyarrow_files pyarrow==0.12.0-RAY \
-      --find-links https://s3-us-west-2.amazonaws.com/arrow-wheels/bf0f3a15e0d6583cfaedbca508627b06746fa41a/index.html
+      --target=$ROOT_DIR/python/ray/pyarrow_files pyarrow==0.12.0.RAY \
+      --find-links https://s3-us-west-2.amazonaws.com/arrow-wheels/9357dc130789ee42f8181d8724bee1d5d1509060/index.html
```

Comment:
Is `9357dc130789ee42f8181d8724bee1d5d1509060` an actual Arrow commit? I don't have a way of knowing what branch or where this comes from.

Reply:
it is a commit of https://github.com/ray-project/arrow-build (right now it's the current master), which contains the arrow commit; this is described in the comment above

Reply:
Ok, in the future, I think it'd be preferable to make the S3 folder have the same name as the arrow commit instead of the arrow-build commit.